### DISCUSSION TITLE
add reload themes to quickbar

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#10990] [Dashboard]
would be very handy to have. currently we need to either run a script constantly, or even worse, go to the service page.

hit c, theme would be awesome

**Comment by GrizzlyAK** [comment]
+10 :grin:

**Comment by spacegaier** [comment]
I agree and I already have it working locally in my dev instance, that needs a few tweaks, then I will push a PR.

EDIT: https://github.com/home-assistant/frontend/pull/11072

**Comment by Mariusthvdb** [comment]
Philip, while you're at it, could you also add the Customizations? It's the other entry not available to the quickbar, and very useful, especially now we can only add customizations via Yaml.

we had https://github.com/home-assistant/frontend/discussions/10067 but 'Core' cant be reloaded from the quickbar either, so I never really understood what Bram said there.

**Reply by spacegaier** [reply]
Part of the same PR

**Comment by Mariusthvdb** [comment]
this has been added... ;
